.rhcicd/stage-post-deployment-tests.yaml

```diff
@@ -23,4 +23,4 @@ parameters:
 - name: UID
   description: "Unique CJI name suffix"
   generate: expression
-  from: "[a-z0-9]{6}"
\ No newline at end of file
+  from: "[a-z0-9]{6}"
```

Comment:
I think we need the last empty line (not sure, but I've seen deployments failing on this)

Reply:
added